modules/fluid_properties/src/userobjects/SodiumProperty.C

```diff
@@ -0,0 +1,117 @@
+/*************************************************/
+/*           DO NOT MODIFY THIS HEADER           */
+/*                                               */
+/*                     BISON                     */
+/*                                               */
+/*    (c) 2015 Battelle Energy Alliance, LLC     */
+/*            ALL RIGHTS RESERVED                */
+/*                                               */
+/*   Prepared by Battelle Energy Alliance, LLC   */
+/*     Under Contract No. DE-AC07-05ID14517      */
+/*     With the U. S. Department of Energy       */
+/*                                               */
+/*     See COPYRIGHT for full restrictions       */
+/*************************************************/
+
+#include "SodiumProperty.h"
+#include "MooseError.h"
+
+template <>
+InputParameters
+validParams<SodiumProperty>()
+{
+  InputParameters params = validParams<FluidProperties>();
+  params.addClassDescription("Fluid properties for sodium");
+  return params;
+}
+
+SodiumProperty::SodiumProperty(const InputParameters & parameters) : FluidProperties(parameters) {}
+
+Real
+SodiumProperty::thermal_conductivity(Real T) const
+{
+  // From page 181. Valid from 371 to 1500 K.
```

Comment:
Move these report references + valid range comments into `.h`, they will be part of the generated doco and thus more useful.